esp/src/src-react/themes.ts

```diff
@@ -33,6 +33,14 @@ const lightThemeOld: PartialTheme = {
         neutralDark: "#222222",
         black: "#000000",
         white: "#ffffff",
+    },
+    semanticColors: {
```

Comment:
This could possibly be removed from the PR. It was in the branch that I had started several months ago, and so I just left it in. I thought I remembered these being used elsewhere, Icons or warning labels or something. But I can't find any references.